src/content/docs/en/guides/assets.mdx

```diff
@@ -105,6 +105,12 @@ Your existing images in `public/` and remote images are still valid but are not
 
 **Built-in asset support is incompatible with the `@astrojs/image` integration.**
 
+:::note
+  Please note that the Assets API doesn't yet support cropping (the current behaviour is to resize images to fit).
+  
+  For now, please continue using the `@astrojs/image` integration if you need support for cropping images. Alternatively you can use the [`object-fit`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) CSS property to achieve a similar result. 
```

Comment:
Thanks for the contribution @eyrewiut, I made a few changes for you to consider!

1. removed unnecessary whitespace
2. changed "behaviour" to "behavior" (we default to the American spelling in docs)
3. added the missing comma after "Alternatively"
4. made the link more explicit by adding the "CSS property" as part of the it.

```suggestion
Please note that the Assets API doesn't yet support cropping (the current behaviour is to resize images to fit).

For now, please continue using the `@astrojs/image` integration if you need support for cropping images. Alternatively, you can use the [`object-fit` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) to achieve a similar result. 
```